ci/parse-trivy-results.py

```diff
@@ -0,0 +1,124 @@
+"""Script to parse results from Trivy scan.
+
+1) Reads in results from two files (json and table format) generated by Trivy.
+2) Creates a GitHub issue in the given repo if any vulnerabilities were found.
+3) Fails the CI build if any vulerabilities of HIGH/CRITICAL severity exist.
+
+Both the json and table Trivy results must be created prior to running
+this script. These can be generated with:
+    trivy image --format json --light --no-progress -o <trivy_output>.json
+        gcr.io/cre-tools/ts-bridge
+    trivy image --light  --no-progress -o <trivy_output>.table
+        gcr.io/cre-tools/ts-bridge
+
+Usage:
+    python3 parse-trivy-results.py <trivy_output> <commit_id> <build_id> <release_tag> <repo_name> <token_file>
+
+Example usage:
+    python3 parse-trivy-results.py trivy-out 99637b2 c933bec7-9acd-48e2-bd7f-7ed7251b5792 1.1.1 google/ts-bridge git_token.txt
+
+"""
+import json
+import sys
+from github import Github
+
+CMDLINE_ARGS = 7
+
+def load_results():
+    """ Load the results from Trivy."""
+    trivy_out_json = "{}.json".format(sys.argv[1])
+    trivy_out_table = "{}.table".format(sys.argv[1])
+    print(trivy_out_table, trivy_out_json)
```

Comment:
Give a bit more details on what's printing here so it's easier to understand what's going on in the build log

Reply:
This was a debug message which I forgot to remove. I have added comments in the lines following to provide more context to trivy results